plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -193,9 +193,18 @@ function exportSubmissions($submissionIds, $context, $user) {
 			$submission = $submissionDao->getById($submissionId, $context->getId());
 			if ($submission) $submissions[] = $submission;
 		}
+		libxml_use_internal_errors(true);
 		$submissionXml = $exportFilter->execute($submissions);
```

Comment:
I noticed on the OJS commits that the execute call gets a second argument, set to `true`. Should that be the case here as well?